src/components/attendee/SpaceInfo/SpaceInfo.module.scss

```diff
@@ -0,0 +1,39 @@
+@use "scss/attendee/font";
+@use "scss/attendee/theme";
+@use "scss/attendee/space";
+@use "scss/attendee/layout";
+@use "scss/attendee/breakpoint";
+
+.spaceWrapper {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  max-width: layout.$w-overlay--max;
+  min-height: 100vh;
+  margin: space.empty(10) auto 0 0;
+  padding: space.empty(4) space.empty(3);
+
+  @media (max-width: breakpoint.$md) {
+    margin: 0;
+  }
+}
+
+.spaceImage {
+  width: 188px;
+  height: 188px;
```

Comment:
[Q] Any reason why these exact values? Is the SVG image that dimensions? Could be useful to have a comment explaining

Reply:
it's the dimensions used in the design, changed to constants now
[a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)